setup.py

```diff
@@ -0,0 +1,46 @@
+from setuptools import setup, find_packages
+import os
+import io
+
+SETUP_DIR = os.path.dirname(os.path.abspath(__file__))
+
+# List all of your Python package dependencies in the
+# requirements.txt file
+
+def readfile(filename, split=False):
+    with io.open(filename, encoding="utf-8") as stream:
+        if split:
+            return stream.read().split("\n")
+        return stream.read()
+
+readme = readfile("README.rst", split=True)[3:]  # skip title
+# For requirements not hosted on PyPi place listings
+# into the 'requirements.txt' file.
+requires = [
+    # minimal requirements listing
+    #'opencmiss.utils @ https://api.github.com/repos/OpenCMISS-Bindings/opencmiss.utils/tarball/master',
+    #'scipy',
+    #'numpy',
```

Comment:
These are commented out but shouldn't be.

Reply:
Actually only need opencmiss.utils (now that I'll actually use it).
Can I put 'opencmiss.zinc' in this list even if requires non-pypi installation? Will it install OK if opencmiss.zinc is already found?

Reply:
If it is already found then the requirement is already satisfied so no problem.  As `opencmiss.zinc` is not yet available on PyPi it is probably better to add it in commented out for now.